pkg/rhRegion/rhRegion.go

```diff
@@ -0,0 +1,60 @@
+package rhRegion
+
+import (
+	json2 "encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+type Region struct {
+	URL string
+	AWS []string
+	GCP []string
+}
+
+func GetRhRegions(ocmServiceUrl string) (map[string]Region, error) {
+	var regions map[string]Region
+	url, err := DetermineRegionDiscoveryUrl(ocmServiceUrl)
+	if err != nil {
+		return regions, fmt.Errorf("Can't determine region discovery URL: %s\n", err)
+	}
+	resp, err := http.Get(url)
+	if err != nil {
+		return regions, fmt.Errorf("Can't retrieve shards: %s\n", err)
+	}
+	err = json2.NewDecoder(resp.Body).Decode(&regions)
+	if err != nil {
+		return regions, fmt.Errorf("Can't decode shards: %s\n", err)
+	}
+	return regions, nil
+}
+
+func GetRhRegion(ocmServiceUrl string, regionName string) (Region, error) {
+	regions, err := GetRhRegions(ocmServiceUrl)
+	if err != nil {
+		return Region{}, err
+	}
+	regionName = fmt.Sprintf("rh-%s", regionName)
+	regVal, ok := regions[regionName]
+	if !ok {
+		return Region{}, fmt.Errorf("Can't find region %s\n", regionName)
+	}
+	return regVal, nil
+}
+
+func DetermineRegionDiscoveryUrl(ocmServiceUrl string) (string, error) {
+	baseUrl, err := url.Parse(ocmServiceUrl)
+	if err != nil {
+		return "", err
+	}
+	regionDiscoveryHost := "api.openshift.com"
+	if strings.HasSuffix(baseUrl.Hostname(), "integration.openshift.com") {
+		regionDiscoveryHost = "api.integration.openshift.com"
+	} else if strings.HasSuffix(baseUrl.Hostname(), "integration.openshift.com") {
+		regionDiscoveryHost = "api.stage.openshift.com"
+	}
+	regionDiscoveryHost = "api.integration.openshift.com"
```

Comment:
this temporary while we play in integration? add a `//TODO: remove before merge` if so

Reply:
Added TODO